caggs on joins (#1992)
timescale/docs: timescaledb/how-to-guides/continuous-aggregates/create-a-continuous-aggregate.md

</procedure>

## Create a continuous aggregate on JOINed tables

> 

Reviewer: I am finding this term JOINed tables a bit odd/unclear, excuse me if it is a general term used by customers or user community. I would have preferred following wording for this, perform joins in the definition of continuous aggregates, or perform joins while creating/defining continuous aggregates, something like that.

Reviewer 2: Agreed, I think I'd prefer `Create a continuous aggregate with a JOIN in the definition`

Author: I fiddled with the wording a bit. Is it clearer now?

In TimescaleDB 2.9 and later, with PostgreSQL 13 or later, you can create a

> continuous aggregate on tables that are connected with a `JOIN`. For example:

Reviewer 2: ```suggestion
continuous aggregate from a query that uses a `JOIN`. For example:
```

Is this better? The underlying table itself is not permanently connected with a `JOIN`, it's the query (and now the continuous aggregate) that is, so maybe the wording is a bit less confusing? Will bow to the expertise of someone who knows SQL better though
